frontend/packages/kubevirt-plugin/integration-tests/views/kubevirtDetailView.view.ts

```diff
@@ -0,0 +1,51 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { resourceRows } from '../../../../integration-tests/views/crud.view';
+
+export const createNic = $('#create-nic-btn');
+export const createDisk = $('#create-disk-btn');
+
+export const nicName = $('#nic-name');
+export const macAddress = $('#mac-address');
```

Comment:
on my build the id's changed, should we update here ?
```
export const macAddress = $('#nic-mac-address');
export const networkTypeDropdownId = '#nic-network-type';
export const networkBindingId = '#nic-binding';
```

Reply:
Thanks for pointing this out, updated